mindjunkies/forums/tests/test_views.py

```
@@ -0,0 +1,369 @@
+    def test_post_valid_comment(self):
+        """Test comment submission with valid data"""
+        request = self.factory.post('/', {'content': 'Test comment content'})
+        request.user = self.user
+        
+        # Add messages support to request
+        setattr(request, 'session', 'session')
+        messages = FallbackStorage(request)
+        setattr(request, '_messages', messages)
+        
+        view = CommentSubmissionView.as_view()
+        response = view(
```

Ruff (0.8.2)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)